decentraland/unity-renderer
# fix: UiShape now refreshes when screen size changes

## What does this PR change?

Fixes #814 

...

## How to test the changes?

Unity Only:
1. Change branch to `fix/screen-resize-uishape`
2. Start the game with _**Open Browser When Start**_ toggled **OFF**
2. Create a new scene with `dcl init`
3. Add this to game.ts
```typescript
const canvas = new UICanvas()
const board = new UIContainerRect(canvas)
board.name = "board-rect"
board.width = "60%"
board.height = "60%"
board.hAlign = "center"
board.color = Color4.Green()
```

4. Start that scene with `dcl start` then add `&ws=ws%3A%2F%2Flocalhost%3A4999%2Fdcl` to the link
5. Resize the Game view, the UI should adapt to the new screen size
